templates/cinder/bin/run-outside

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec nsenter -a -t 1 -- `realpath -s $0` "$@"
```

Comment:
Yes, `realpath` is important, because the $PATH could be different in the host and the container, so a binary that could be found in the container via $PATH may not be found on the host.   Using `realpath` resolves that issue for us.

I wrote the `run-outside` script 2 years ago [1], and to be honest I don't recall the reason behind enclosing it in double quotes.

[1]: https://github.com/embercsi/ember-csi/commit/bda33682e73b5299fa176908691272d5751a7377